code/modules/antagonists/eldritch_cult/eldritch_antag.dm

```diff
@@ -22,21 +29,21 @@
 	to_chat(owner, "<span class='boldannounce'>You are the Heretic!</span><br>\
 	<B>The old ones gave you these tasks to fulfill:</B>")
 	owner.announce_objectives()
-	to_chat(owner, "<span class='cult'>The book whispers, the forbidden knowledge walks once again!<br>\
-	Your book allows you to research abilities, read it very carefully! You cannot undo what has been done!<br>\
-	You gain charges by either collecting influences or sacrificing people tracked by the living heart<br> \
+	to_chat(owner, "<span class='cult'>As a heretic, you can gain the favor of the old gods through various acts to enhance your arcane powers<br>\
+	You can start by seeking out strange figurines to gain the favor of the lesser gods.<br>\
+	When you are ready, carve a Rune of Transmutation and transmute a Codex Cycatrix from a bible, a pair of eyes and a pen, to unlock your true powers.<br> \
```

Comment:
a pair of eyes seems like it would be pretty hard to get especially since they don't get their powers until they get one.

Reply:
Yea I've given that some thought.

I could either make them start with codex again, or make it cheaper to craft (since it's not as crucial to make for heretics).